server/app.js

```diff
@@ -43,22 +43,52 @@ app.get('/login', (req, res) => {
 
 
 
+app.post('/state', (req, res)=>{
+    let output = {
+        success: false,
+        data: null,
+        errors: null
+    };
+
+    console.log(req);
+
+
+    const query = `CALL getUserStateDetails(${req.user.id}, ${req.body.story})`;
+
+    connection.query(query, function(error, data){
+
+        if (!error){
+            let formattedData = formatStateData(data);
+
+            output.success = true;
+            output.data = formattedData;
+        } else {
+            output.errors = error; // will change this when no longer testing
+        }
+
+        res.send(output);
+    });
+
+});
+
 app.post('/action', (req, res) => {
```

Comment:
This whole route should be moved to auth.js and have the isLoggedIn function called on it. We arent wanting people to request a post to this path without being logged in, right?